avocado/utils/data_structures.py

```diff
@@ -33,6 +34,24 @@ def ordered_list_unique(object_list):
     return [x for x in object_list if not (x in seen or seen_add(x))]
 
 
+def geometric_mean(values):
+    """
+    Evaluates the geometric mean for a list of numeric values.
+    This implementation is slow.
```

Comment:
I wouldn't complain, but as you need to update the value, would you please also either remove this line or use `This implementation is slower but allows unlimited number of values`.